VRoidHubLoader/Modules/VrmLoaderModule.cs

```diff
@@ -62,17 +63,37 @@ private void Update()
         {
             Core.Msg("OnUpdate: VrmLoaderModule F4 pressed");
 
-            // MenuManager is a singleton? sweet.
-            if (!MenuManager.Instance.IsOpen)
+            await ImportVRM();
+        }
+    }
+
+    [HideFromIl2Cpp]
+    public async Task ImportVRM()
+    {
+        string? path = await Core.FileHelper.OpenFileDialog();
+
+        if (!string.IsNullOrEmpty(path))
+        {
+            string fileName = Path.GetFileName(path);
+            string destination = VrmFolderPath + '\\' + fileName;
+            if (!File.Exists(destination))
             {
-                #if MELON
-                MelonCoroutines.Start(CoAutoOpenModelPage());
-                #endif
-                #if BEPINEX
-                MonoBehaviourExtensions.StartCoroutine(this, CoAutoOpenModelPage());
-                #endif
+                File.Copy(path, destination);
```

Comment:
Just another thought, what if this was a symbolic link? That would avoid extra storage space being taken up, especially with bigger 100MB+ models, but it would be less stable and we'd probably need some cleanup logic to delete broken symbolic links. Maybe this is a bigger thing, but might be necessary if we go the suffixing route with duplicate models being added since otherwise the VRM folder might grow a bit too quickly.

Reply:
I never thought about doing it that way, but in my opinion as of now we don't really have to worry too much about it. We can definitely have this on the backburner but for now I think making a copy or simply moving the file to the VRM folder will suffice.

Reply:
Yeah, very true, this is more of a potentially interesting thing to look into in the future if we're bored.